x-pack/legacy/plugins/apm/public/components/shared/Summary/HttpStatusBadge/__test__/HttpStatusBadge.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { HttpStatusBadge } from '../index';
+
+describe('HttpStatusBadge', () => {
+  it('with status code 100', () => {
```

Comment:
I'd break these up further, mostly because I don't like that `it('with status code 100'` makes no sense if you read it out loud. I would make it:

```js
describe('HttpStatusBadge', () => {
  describe('render', () => {
    describe('with status code 100', () => {
      it('renders with the darkShade color', () => {...}
```

it's more verbose but reads better and looks nicer in the output.